strongbox-user-management/src/main/java/org/carlspring/strongbox/authentication/api/impl/ldap/LdapAuthenticator.java

```diff
@@ -0,0 +1,29 @@
+package org.carlspring.strongbox.authentication.api.impl.ldap;
+
+import org.carlspring.strongbox.authentication.api.Authenticator;
+
+import javax.annotation.Nonnull;
+import javax.inject.Inject;
+
+import org.springframework.security.authentication.AuthenticationProvider;
+import org.springframework.security.ldap.authentication.LdapAuthenticationProvider;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+@Component
+public class LdapAuthenticator
```

Comment:
@fuss86 what do you think about [this](https://github.com/strongbox/strongbox/pull/272#discussion_r120262250)? On my opinion it can be a good feature.